src/VCR_InterfaceTasks.cpp

```diff
@@ -48,8 +64,10 @@ bool init_read_adc1_task()
     float scales[channels_within_mcp_adc] = {1, 1, 1, 1, 1, 1, 1, 1};
     float offsets[channels_within_mcp_adc] = {0, 0, 0, 0, 0, 0, 0, 0};
     /* NOLINTEND */
+    static MCP_ADC<channels_within_mcp_adc> adc0(ADC0_CS, MCP_ADC_DEFAULT_SPI_SDI, MCP_ADC_DEFAULT_SPI_SDO, MCP_ADC_DEFAULT_SPI_CLK, MCP_ADC_DEFAULT_SPI_SPEED, scales, offsets);
+    static MCP_ADC<channels_within_mcp_adc> adc1(ADC1_CS, MCP_ADC_DEFAULT_SPI_SDI, MCP_ADC_DEFAULT_SPI_SDO, MCP_ADC_DEFAULT_SPI_CLK, MCP_ADC_DEFAULT_SPI_SPEED, scales, offsets);
```

Comment:
Why are we calling create() again here? You only need to instantiate the bundle once. It's a singleton.